website/blog/2021-06-24-faster-database-imports-in-studio.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "WaveMaker Studio enables faster database imports"
+author: Sanjana Raheja
+---
+
+WaveMaker platform offers the feature for applications to integrate with external databases like MySQL, SQLServer, Oracle, IBM DB2, PostgreSQL, Amazon Redshift, SAP HANA.
+ 
+In the application, the developer can configure the database connection settings during database import operation. Post this, the application developer is provided to choose from the list of tables and constraint information present in the database to proceed with the database import operation.
+
+We have optimised this flow to provide an enriched performance to the end user. 
+
+<!-- truncate -->
+
+
+### How did we enable faster database imports in Studio?
+
+When a user imports a database into Studio, there are a couple of queries that are run in background to fetch the table and constraints information from the system tables in the database. We have optimised these queries at our end to enable faster database imports in Studio.
```

Comment:
We should explain that we read the schema and discover relationship between the tables and individual table schema as well. Armed with this information WM generates REST API and create hibernate entities for each of tables in the database.